kolibri/core/negotiation.py

```diff
@@ -1,21 +1,29 @@
-from rest_framework.negotiation import BaseContentNegotiation
+from rest_framework.negotiation import DefaultContentNegotiation
 from rest_framework.parsers import JSONParser
 from rest_framework.parsers import MultiPartParser
 from rest_framework.renderers import JSONRenderer
 
 
-class LimitContentNegotiation(BaseContentNegotiation):
+class LimitContentNegotiation(DefaultContentNegotiation):
     def select_parser(self, request, parsers):
         """
         Always return JSONParser unless a 'multipart/form-data' content type is sent
+        or this is for morango
         """
+
+        if request.path.startswith("/api/morango"):
+            return super(LimitContentNegotiation, self).select_parser(request, parsers)
         if "multipart" in request.content_type:
             return MultiPartParser()
         return JSONParser()
 
-    def select_renderer(self, request, renderers, format_suffix):
+    def select_renderer(self, request, renderers, format_suffix=None):
```

Comment:
Does it need the renderer? Because it's gzipping the output too?

Reply:
Hmm, I don't think it's needed.

Reply:
So, do we need the special case for morango in the renderer, or just need the parsing change above?